runtime/kusama/src/weights/pallet_staking.rs

```diff
@@ -201,14 +201,14 @@ impl<T: frame_system::Config> pallet_staking::WeightInfo for WeightInfo<T> {
 	}
 	fn submit_solution_better(v: u32, n: u32, a: u32, w: u32, ) -> Weight {
 		(0 as Weight)
-			// Standard Error: 43_000
-			.saturating_add((1_426_000 as Weight).saturating_mul(v as Weight))
-			// Standard Error: 17_000
-			.saturating_add((588_000 as Weight).saturating_mul(n as Weight))
-			// Standard Error: 43_000
-			.saturating_add((69_239_000 as Weight).saturating_mul(a as Weight))
-			// Standard Error: 90_000
-			.saturating_add((8_062_000 as Weight).saturating_mul(w as Weight))
+			// Standard Error: 46_000
+			.saturating_add((712_000 as Weight).saturating_mul(v as Weight))
```

Comment:
hmm again this weight is almost halved. I think this means that the weight calculation was not accurate enough, not an issue with this PR.